mainwindow.cc

```diff
@@ -3105,7 +3105,12 @@ void MainWindow::hotKeyActivated( int hk )
     toggleMainWindow();
   else
   if ( scanPopup.get() )
+#ifdef HAVE_X11
+    // workaround of clipboard issue with Ctrl+C+C
```

Comment:
The comment is a bit too short. Could you explain the worked-around issue in more detail?